schema.json

```diff
@@ -248,6 +248,11 @@
             "description": "e.g. 1989-06-12",
             "format": "date"
           },
+          "url": {
```

Comment:
Adding a `url` to the `awards` section is a separate issue. Remove this.

Reply:
`url` has been removed from `awards` section.